executor/analyze.go

```diff
@@ -1093,16 +1093,16 @@ func (e *AnalyzeFastExec) updateCollectorSamples(sValue []byte, sKey kv.Key, sam
 			}
 			idxVals = append(idxVals, v)
 		}
-		var bytes []byte
-		bytes, err = codec.EncodeKey(e.ctx.GetSessionVars().StmtCtx, bytes, idxVals...)
+		var keyBytes []byte
+		keyBytes, err = codec.EncodeKey(e.ctx.GetSessionVars().StmtCtx, keyBytes, idxVals...)
```

Comment:
The original name `bytes` conflicts with the imported package `bytes`

Reply:
(there are many places naming a variable as `byte`.:joy:

Reply:
Well, that's really a bad news :sob: